README.md

```diff
@@ -49,24 +51,17 @@ $ npm install
 Simplest method, will do everything needed in one command.
 
 ```bash
-$ bash artifact.sh
+$ source artifact-lang.sh
```

Comment:
To be able to retreive theses files, a github personal token is mandatory (env GIT_TOKEN)
See https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token